components/Footer.js

```diff
@@ -1,40 +1,86 @@
 import Link from 'next/link';
 import React from 'react';
+import styles from '../styles/components/Footer.module.scss';
 import SocialMedia from './SocialMedia';
 
+// TODO: add GM?
+const footerACMLinks = [
+	{ title: 'About', path: '/about' },
+	{ title: 'Committees', path: '/committees' },
+	{ title: 'Events', path: '/events' },
+	{ title: 'Sponsors', path: '/sponsors' },
+	{ title: 'Tech Gala', path: '/techgala' },
+	{ title: 'Membership Portal', path: 'https://members.uclaacm.com', ext: true },
+];
+
+// Do we want to prepend ACM here?
+const footerCommitteeLinks = [
+	{ title: 'AI', path: 'https://uclaacmai.github.io/', ext: true },
+	{ title: 'Cyber', path: 'https://acmcyber.com', ext: true },
+	{ title: 'Design', path: '/committees#design' },
+	{ title: 'Hack', path: 'https://hack.uclaacm.com', ext: true },
+	{ title: 'ICPC', path: 'https://icpc.uclaacm.com', ext: true },
+	{ title: 'Studio', path: 'http://localhost:3000/committees#studio' },
+	{ title: 'Teach LA', path: 'https://teachla.uclaacm.com', ext: true },
+	{ title: 'W', path: '/committees#w' },
```

Comment:
How did you settle on these links? If I recall correctly, we have some committee data stored in objects in `/data/committees.js`. 

Do we want to standardize the links we pull somehow. The nav gets a little tricky with the footer committee links since many of lead to external sites while the links in our `CommitteeSpread` component on the landing page all route to the `/committee` page.